app/routes/events/$slug.tsx

```diff
@@ -18,13 +19,44 @@ export const loader: LoaderFunction = async ({ params }) => {
 
 export default function EventSlug() {
   const event = useLoaderData<typeof loader>();
-
+  const actualDate = new Date(event.frontmatter.date);
   return (
-    <div>
-      <main>
-        <h1>Event</h1>
-        <img src={event.frontmatter.front_image} />
-      </main>
+    <div className="eventLG__content">
+      <div className="eventLG__text_container">
+        <h3 className={"eventLG__text_large"}>{event.frontmatter.title}</h3>
+        <p className={"eventLG__text_medium"}>
```

Comment:
⛏️ Since your just using strings here for classnames you don't need to send them through the curly braces for string interpolation. Here you can just drop the {}'s and still be fine.

Reply:
oops, I copied the code from the event component and forgot to remove those. I made another commit without them.